sphinx/source/docs/dev_guide/server.rst

```diff
@@ -258,8 +259,8 @@ two sequences of frames, one sequence in each direction ("full duplex").
 On top of websocket frames, we implement our own ``Message`` concept. A Bokeh
 ``Message`` spans multiple websocket frames. It always contains a header frame,
 metadata frame, and content frame. These three frames each contain a JSON
-string. The code permits these three frames to be followed by binary data
-frames, but currently in Bokeh binary data frames are not used.
+string. The code permits these three frames to be followed by optional binary data
+frames. Currently in Bokeh, binary data frames are only used to transport image data.
```

Comment:
Thanks @bryevdv and @philippjfr.   When I was tweaking the docs, I wondered if the recent work was base64 encoding the data into the content block or actually using 'binary' frames.  Now that I have a pointer to @philippjfr's work in #5544, I'll read through his code changes and take another pass at this section of the dev_guide.